Added enheder til forskelllige shops

DominicNr2 opened the pull request:
titlen jaer(jæger)

DominicThai committed:
Added enehder til forskelllige shops

titlen jaer

mathiasgredal committed:
Merge branch 'master' into Add_Units_To_Price

mathiasgredal commented on src/main/java/worldofzuul/Items/Buyable.java:
```diff
@@ -25,4 +25,5 @@
      *
      * Gets the output in kWh for the item
      */
+
```
Why is there an empty javadoc comment here

mathiasgredal reviewed (changes requested):
Almost looks good

Nolle10 committed:
Made the price change every year, the moneyLabel position is changed.

I also did so you can enter the house after buying a heater and/or an electric car

Nolle10 committed:
Fixed the empty JavaDoc comment

Nolle10 committed:
Merge branch 'master' into Add_Units_To_Price

Nolle10 commented:
I do be fixing merge conflict 🐒🦧🦍🐵

Nolle10 committed:
Merge branch 'master' into Add_Units_To_Price

Nolle10 committed:
Optimized the code, by deleting some unused imports

L337 PR0G4MM3R

Nolle10 committed:
Merge branch 'master' into Add_Units_To_Price

mathiasgredal reviewed (approved):
LGTM
